t/apicast-policy-rate-limit.t

```diff
@@ -1313,3 +1313,62 @@ limit. The limit is set to 2. Only the third one should fail.
 [200, 200, 429]
 --- no_error_log
 [error]
+
+
+=== TEST 22: Window is not set and default to 1
+Return 429 code.
+--- http_config
+  include $TEST_NGINX_UPSTREAM_CONFIG;
+  lua_package_path "$TEST_NGINX_LUA_PATH";
+
+  init_by_lua_block {
+    require "resty.core"
+    ngx.shared.limiter:flush_all()
+    require('apicast.configuration_loader').mock({
+      services = {
+        {
+          id = 42,
+          proxy = {
+            policy_chain = {
+              {
+                name = "apicast.policy.rate_limit",
+                configuration = {
+                  fixed_window_limiters = {
+                    {
+                      key = {name = "test9", scope = "global"},
+                      count = 1,
+                      window = 0
+                    }
+                  },
+                  redis_url = "redis://$TEST_NGINX_REDIS_HOST:$TEST_NGINX_REDIS_PORT/1",
```

Comment:
Optional: if you don't define this limit as global, you will not need redis, and you'll be able to simplify a bit the test. You'll be able to delete this line, but also the `/flush_redis` location and the first request in the test.